packages/owl-bot/cloudbuild.yaml

```diff
@@ -117,12 +117,18 @@ steps:
     env:
       - "IMAGE_NAME=gcr.io/$PROJECT_ID/owl-bot-backend"
       - "SERVICE_NAME=owl-bot-backend"
-      - "MEMORY=1G"
+      - "MEMORY=4G"
```

Comment:
We have so many cloud build yaml files that I lose track.  Please add a comment to the top of this file describing when and why this cloud build is invoked, and its output.

Reply:
Done, PTAL